include/functions.h

```diff
@@ -877,9 +877,9 @@ f32 Math_SmoothStepToDegF(f32* pValue, f32 target, f32 fraction, f32 step, f32 m
 s16 Math_SmoothStepToS(s16* pValue, s16 target, s16 scale, s16 step, s16 minStep);
 void Math_ApproachS(s16* pValue, s16 target, s16 scale, s16 step);
 void Color_RGBA8_Copy(Color_RGBA8* dst, Color_RGBA8* src);
-void func_80078884(u16 sfxId);
-void func_800788CC(u16 sfxId);
-void func_80078914(Vec3f* arg0, u16 sfxId);
+void Lib_PlaySfx(u16 sfxId);
+void Lib_PlaySfx_Centered(u16 sfxId);
+void Lib_PlaySfx_AtPos(Vec3f* projectedPos, u16 sfxId);
```

Comment:
personally dont think the extra underscore adds much and would be fine with 
`Lib_PlaySfx`
`Lib_PlaySfxCentered`
`Lib_PlaySfxAtPos`

but yea should get more opinions

Reply:
Looking ahead at the amount of playsfx wrappers though, I understand this choice to tie them all together

Reply:
Look at the PR description (opening comment) (at least that's what I refer to by what I looked ahead at)

Reply:
I see
what about making the lib functions just have a system name of `Sfx_`? then the function itself can just be `Sfx_PlayReallyLongExplanation`

lib in general doesnt really mean that much as far as system goes. and if this route is taken, I think the z_actor ones can afford to also live without the subsystem scheme cause they arent that long

Reply:
`Actor_PlaySfx` wrappers all take an `Actor*` so they're probably better off using the `Actor_` system name

for the three `Lib_PlaySfx` wrappers, idk what else they could use
`Audio_` or `AudioSfx_` (instead of `Lib_`) could be tempting but probably we should reserve that to the "external audio" code (see #1494 , `audio_external_sfx.c`)

could coin `Sfx_` just for these three yeah :shrug: not worse than `Lib_`

Reply:
Apologies for the delayed response on my part, been meaning to think about the whole sfx naming scheme if we're to avoid the double underscores.

`Sfx_` could work, but then having two `Sfx` in `Sfx_PlaySfx` looks a bit weird to me, but maybe that's fine

Reply:
`Lib_` is completely meaningless so yeah `Sfx_PlaySfx` sounds fine :shrug:

Reply:
for functions with sfx in the system name, I dont totally hate `Sfx_Play` `Sfx_PlayAtPos` etc
I think this is even the suggestion i made above a couple months ago. but yeah this is fine too i suppose

Reply:
Seems fine to me as well :)